tests/sqlparser_mssql.rs

```diff
@@ -192,6 +192,35 @@ fn parse_mssql_apply_join() {
     );
 }
 
+#[test]
+fn parse_mssql_cross_apply_json() {
+    let _ = ms().verified_only_select(
+        "SELECT B.kind, B.id_list \
+        FROM t_test_table AS A \
+        CROSS APPLY OPENJSON(A.param, '$.config') WITH (kind VARCHAR(20) '$.kind', [id_list] NVARCHAR(MAX) '$.id_list' AS JSON) AS B",
+    );
```

Comment:
```suggestion
    ms_and_generic().verified_only_select(
        "SELECT B.kind, B.id_list \
        FROM t_test_table AS A \
        CROSS APPLY OPENJSON(A.param, '$.config') WITH (kind VARCHAR(20) '$.kind', [id_list] NVARCHAR(MAX) '$.id_list' AS JSON) AS B",
    );
```

Reply:
`GenericDialect` doesn't support brackets around the identifier.